components/sysutils/gnupg/manifests/sample-manifest.p5m

```diff
@@ -148,4 +149,7 @@ file path=usr/share/man/man1/watchgnupg.1
 file path=usr/share/man/man1m/addgnupghome.1m
 file path=usr/share/man/man1m/applygnupgdefaults.1m
 file path=usr/share/man/man1m/dirmngr.8
+file path=usr/share/man/man1m/man8/addgnupghome.8
```

Comment:
When I generate a sample-manifest the man8 files are missing:
> file path=usr/share/man/man1m/man8/addgnupghome.8
> file path=usr/share/man/man1m/man8/applygnupgdefaults.8
> file path=usr/share/man/man1m/man8/dirmngr.8

The paths man1m/man8 also seem to be wrong!